packages/manager/apps/hub/src/dashboard/incident/status/resiliation-pci/resiliation-pci.module.js

```diff
@@ -0,0 +1,25 @@
+import angular from 'angular';
+import 'angular-translate';
+import '@ovh-ux/ui-kit';
+import uiRouter from '@uirouter/angularjs';
+import ngOvhTranslateAsyncLoader from '@ovh-ux/ng-translate-async-loader';
+import ngUiRouterLayout from '@ovh-ux/ng-ui-router-layout';
+
+import component from './resiliation-pci.component';
+import routing from './routing';
+
+const moduleName = 'ovhManagerHubIncidentStatusResiliationPci';
+
+angular
+  .module(moduleName, [
+    ngOvhTranslateAsyncLoader,
+    'oui',
+    ngUiRouterLayout,
+    'pascalprecht.translate',
+    uiRouter,
```

Comment:
In many place of directories we use this style:

```suggestion
  .module(moduleName, [
    ngOvhTranslateAsyncLoader,
    ngUiRouterLayout,
    uiRouter,
    'oui',
    'pascalprecht.translate',
```

Reply:
Yes to sort alphabetically but should be done automatically if mandatory